### PR TITLE
540 auth error convertion

### LINKED ISSUE #540: Authorization error is not converted to an internal error struct
Conversion code is missed: https://github.com/qoollo/bob/blob/master/bob-common/src/error.rs#L143

### PR DESCRIPTION
**Pull request by Boneyan**
Added conversion of "unauthorized" status into internal error 
Closes #540

### CODE REVIEW

File: bob-common/src/error.rs [hunk 1]
```diff
@@ -156,6 +156,7 @@ impl From<Status> for Error {
                 "Failed" => Some(Self::failed(rest_words(words, length))),
                 "Internal" => Some(Self::internal()),
                 "PearlChangeState" => Some(Self::pearl_change_state(rest_words(words, length))),
+                "Unauthorized" => Some(Self::internal()),
```
Comment (ikopylov) [inline bob-common/src/error.rs:159]: Seems like `internal` is a bad choice. Probably, separate error kind should be used for authorization errors

**Comment by ikopylov** [comment]
@Boneyan please, run the test when `Unauthorized` error generated during the work, and check that everything works normally (no panic, data is written to aliens when this error occured)

**Review by ikopylov — APPROVED** [review]
LGTM
Need a test

**Comment by Boneyan** [comment]
I ran the test, all works fine

**Pull request merged**